src/main/java/org/jenkinsci/plugins/codesonar/models/CodeSonarAlertFrequencies.java

```diff
@@ -0,0 +1,70 @@
+package org.jenkinsci.plugins.codesonar.models;
+
+/**
+ * @author aseno
+ *
+ */
+public class CodeSonarAlertFrequencies {
+    private int red;
+    private int yellow;
+    private int blue;
+    private int green;
```

Comment:
Could we drive this "Frequencies" class with an enum of "CodeSonarAlertColors" (or "CodeSonarAlertLevels")?  Then, instead of `getRed()` we would have `getAlertCount(CodeSonarAlertLevel level)`.

I personally prefer the term "level" over "color" since I see alert colors as analogous to log levels, but either term is okay.  I think the level name would still have to be the color name, so introducing the term "level" might not make sense to everyone.

Reply:
I see that there is a `CodeSonarAlertData.Color` enumeration.  Is it possible to move `Color` to the top-level and share it between this class and `CodeSonarAlertData`?